src/components/WizardInline/_wizard-inline.scss

```diff
@@ -0,0 +1,14 @@
+@import '~carbon-components/scss/globals/scss/vars';
+
+.wizard-inline {
+  &-header {
+    font-size: 1.25rem;
```

Comment:
We could use `productive-heading-03` here instead?

Reply:
This will add a change to the line-height as well which I don't think the design called for. At this point I can not find the design I was going off of. I am not opposed adding this mixin.